### PR TITLE
feat: support `libraryReturnUrl` when installing libraries

### PR DESCRIPTION
**Pull request by dwelle**
for https://github.com/excalidraw/excalidraw-libraries/pull/74 (this PR must be merged first due to the URI decoding)

- use `referrer` when loading libs. This isn't necessary as our origin is the default, but it's useful in development
- decode URI component

---

- [x] add prop to Excalidraw component for the host apps
- [x] changelog
- [x] readme

/cc @zsviczian

### CODE REVIEW

File: src/components/LayerUI.tsx [hunk 1]
```diff
@@ -166,7 +166,10 @@ const LibraryMenuItems = ({
         }}
       />
 
-      <a href="https://libraries.excalidraw.com" target="_excalidraw_libraries">
+      <a
+        href={`https://libraries.excalidraw.com?referrer=${window.location.origin}`}
+        target="_excalidraw_libraries"
+      >
```
Comment (dwelle) [inline src/components/LayerUI.tsx:172]: this should actually use a prop from the host app

File: src/packages/excalidraw/CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ Please add the latest change on the top under the correct section.
 
 ### Fixes
 
+- Support `libraryReturnUrl` when installing libraries [3227](https://github.com/excalidraw/excalidraw/pull/3227).
```
Comment (ad1992) [inline src/packages/excalidraw/CHANGELOG.md:21]: can you move this to "Features" ?

File: src/packages/excalidraw/CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,12 @@ The change should be grouped under one of the below section and must contain PR
 Please add the latest change on the top under the correct section.
 -->
 
+## Unreleased
+
+### Features
```
Comment (ad1992) [inline src/packages/excalidraw/CHANGELOG.md:17]:
```suggestion
## Unreleased

## Excalidraw API

### Features
```

File: src/packages/excalidraw/CHANGELOG.md [hunk 4]
```diff
@@ -12,6 +12,12 @@ The change should be grouped under one of the below section and must contain PR
...
+
+- Support `libraryReturnUrl` prop to indicate what URL to install libraries to [#3227](https://github.com/excalidraw/excalidraw/pull/3227).
+
```
Comment (ad1992) [inline src/packages/excalidraw/CHANGELOG.md:20]:
```suggestion
- Support `libraryReturnUrl` prop to indicate what URL to install libraries to [#3227](https://github.com/excalidraw/excalidraw/pull/3227).

---
```

**Review by ad1992 — APPROVED** [review]

**Pull request merged**